### ISSUE TITLE
A question on class VAERes2D_DwT

### ISSUE THREAD

**Issue by zx-bit** [Arlo0o/UniScene-Unified-Occupancy-centric-Driving-Scene-Generation#42]
I have some questions regarding the computation of the KL_Loss. In the original VAE code, the KL_Loss is derived from the following formula:
kld_loss = torch.mean(-0.5 * torch.sum(1 + log_var - mu ** 2 - log_var.exp(), dim = 1), dim = 0)
In VAERes2D_DwT, the KL_Loss is computed as follows:
https://github.com/Arlo0o/UniScene-Unified-Occupancy-centric-Driving-Scene-Generation/blob/e98413de010586cff43dc48c63c998a8e39f3888/occupancy_gen/model_vae/VAE/vae_2d_resnet_DwT.py#L369
Here, the variable z_sigma should represent the log_var. But self.sample_z returns log_var.exp(), which is the standard deviation. 

I would like to inquire whether this observed discrepancy stems from a bug or a misinterpretation on my part. When training a VAERes2D_DwT model, the noise sampled from a standard Gaussian distribution fails to decode into coherent results. Is this phenomenon expected, or does it indicate an issue with the model training?
